Tweak.xm

```diff
@@ -91,6 +103,38 @@ static BOOL shouldShowPicker = NO;
 	}
 }
 
+- (NSString *)activator:(LAActivator *)activator requiresLocalizedTitleForListenerName:(NSString *)listenerName
+{
+    return @"WiPi";
+}
+
+- (NSString *)activator:(LAActivator *)activator requiresLocalizedDescriptionForListenerName:(NSString *)listenerName
+{
+    return @"Show WiFi Picker";
+}
+
+- (NSData *)activator:(LAActivator *)activator requiresSmallIconDataForListenerName:(NSString *)listenerName scale:(CGFloat *)scale
+{
+    if (*scale == 1.0) {
```

Comment:
You dont need to handle scaling of images yourself, iOS will do it automatically if you use the correct UIImage API (it will select the correct @X.png file). Use `[UIImage imageNamed:@"WiPi.png]`, and then use `UIImagePNGRepresentation` to get the NSData representation of that.